grip-mode.el

```diff
@@ -56,6 +56,12 @@
   :type 'file
   :group 'grip)
 
+(defcustom grip-url-browser nil
+  "Browser to launch Markdown/Org previews.
+Use default browser if nil."
+  :type 'string
```

Comment:
Should be: `:type '(choice (const :tag "None" nil) string)`.

Reply:
Ok that makes sense